spec/factories/locations_floors.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :locations_floor, :class => 'Locations::Floor' do
+    label "MyString"
+floor_plan_uri "MyString"
+starting_point "MyString"
+walkable_areas "MyString"
```

Comment:
Can the indentation of lines 4-6 match with line 3? Also could you rename the file to "floors.rb" and move it to "spec/factories/locations," where the rest of the factories are?